open_humans/management/commands/bulk_email.py

```diff
@@ -73,7 +73,7 @@ def handle(self, *args, **options):
         email_file = full_path(options['email_file'])
 
         email_data = []
-        with open(email_file, 'rb') as f:
+        with open(email_file, 'r') as f:
```

Comment:
Curious as to why you are making file reading text-only?  It shouldn't really matter in this case.

Reply:
I was getting this error, which was resolved by it:

`_csv.Error: iterator should return strings, not bytes (did you open the file in text mode?)`